dipy/reconst/mcsd.py

```diff
@@ -439,11 +439,11 @@ def multi_shell_fiber_response(sh_order, bvals, wm_rf, gm_rf, csf_rf,
     bvals : ndarray
         Array containing the b-values. Must be unique b-values, like outputted
         by `dipy.core.gradients.unique_bvals_tolerance`.
-    wm_rf : (4, len(bvals)) ndarray
+    wm_rf : (len(bvals)-1, 4) ndarray
```

Comment:
not sure it is correct `-1`.  maybe `(len(nonzero(bvals)), 4)`. 

I have a doubt here

Reply:
agreed that `(len(nonzero(bvals)), 4)` better communicates the idea.

It seems that `bvals` is expected to have exactly one zero in it, because [it's expected to have been generated by `unique_bvals_tolerance`](https://github.com/dipy/dipy/blob/9153a852a511b07ffc141becfbd9a96ca42e9a90/dipy/reconst/mcsd.py#L441). 

it's allowed for it to not have that zero, but [with mild complaining](https://github.com/dipy/dipy/blob/9153a852a511b07ffc141becfbd9a96ca42e9a90/dipy/reconst/mcsd.py#L520)

what about writing `(N-1, 4) where N is the number of unique b-values including the b0 `
since the docstring later makes this definition with the notation `N`

Reply:
> what about writing (N-1, 4) where N is the number of unique b-values including the b0 
since the docstring later makes this definition with the notation N

I am ok with that 👍

Reply:
How's this?